CMakeLists.txt

```diff
@@ -1190,6 +1190,12 @@ else()
 		ext/disarm.cpp)
 endif()
 
+if (NOT MOBILE_DEVICE)
+	set(CoreExtra ${CoreExtra}
+		Core/AVIDump.cpp
+		Core/AVIDump.h)
```

Comment:
Note: this will get pulled into Qt automatically, so you might need to put `#ifndef MOBILE_DEVICE`.  See here:

https://github.com/hrydgard/ppsspp/blob/master/Qt/Core.pro

(Qt is used for Symbian, for example.)

-[Unknown]